model/model/v1alpha/model.proto

```diff
@@ -1737,3 +1737,92 @@ message UndeployOrganizationModelAdminRequest {
+  // Model run logging status
+  RunStatus status = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model visibility.
+  Model.Visibility visibility = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger source
+  string source = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger start time
+  google.protobuf.Timestamp start_time = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger total duration in milliseconds
+  optional int64 total_duration = 8 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Trigger end time
+  optional google.protobuf.Timestamp end_time = 9 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Trigger requesters
+  string runner = 10 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Credit consumptions
+  optional float credits = 11 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Error msg occurred during model run
+  optional string errors = 12 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Model run logging creation time.
+  google.protobuf.Timestamp create_time = 13 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model run logging update time.
+  google.protobuf.Timestamp update_time = 14 [(google.api.field_behavior) = OUTPUT_ONLY];
+}
+
+// ListModelRunLoggingsRequest represents a request to list of model run loggings
+message ListModelRunLoggingsRequest {
+  // The maximum number of loggings to return. If this parameter is unspecified,
+  // at most 10 loggings will be returned. The cap value for this parameter is
+  // 100 (i.e. any value above that will be coerced to 100).
+  optional int32 page_size = 1 [(google.api.field_behavior) = OPTIONAL];
+  // Page token.
+  optional string page_token = 2 [(google.api.field_behavior) = OPTIONAL];
+  // View allows clients to specify the desired model view in the response.
+  optional View view = 3 [(google.api.field_behavior) = OPTIONAL];
+  // Filter can hold an [AIP-160](https://google.aip.dev/160)-compliant filter
+  // expression.
+  // - Example: `create_time>timestamp("2000-06-19T23:31:08.657Z")`.
+  optional string filter = 4 [(google.api.field_behavior) = OPTIONAL];
+  // Limit results to pipelines with the specified visibility.
+  optional Model.Visibility visibility = 5 [(google.api.field_behavior) = OPTIONAL];
+  // Order by field, with options for ordering by `id`, `create_time` or `update_time`.
+  // Format: `order_by=id` or `order_by=create_time desc`, default is `asc`.
+  optional string order_by = 6 [(google.api.field_behavior) = OPTIONAL];
+}
+
+// ListModelRunLoggingsResponse contains a list of model run loggings.
+message ListModelRunLoggingsResponse {
+  // A list of loggings resources.
+  repeated ModelRunLogging loggings = 1;
```

Comment:
It seems that the log entity (or trigger, or run) will contain more information (input, output) than what's returned in the list endpoint. You'll need either 2 entities (`ModelRunListItem` vs `ModelRun`) or 2 views for the same entity.